static/js/iframe-prod.js

```diff
@@ -1,5 +1,9 @@
 import { generateIFrame } from './iframe-common';
 import InjectedData from './models/InjectedData';
+import { HitchhikerConfigManager, exposeOnWindow } from './hitchhikerconfigmanager';
+
+const hitchhikerConfigManager = new HitchhikerConfigManager();
+exposeOnWindow(hitchhikerConfigManager);
```

Comment:
do the static iframe scripts need hitchhikerConfigManager?

Reply:
ya, I will check with product, but having it here allows the someone with a regular iframe experience to update the config from the parent, even it it doesn't happen during the init